### PR TITLE
Glyph maintenance

### LINKED ISSUE #122: Remove unnecessary unexported glyphs
102468050-ef354200-4048-11eb-8d08-68eb184907f8

---

### TODO

- [x] remove G.ss06
- [x] remove peace icon
- [x] remove whiteFrowningFace
- [x] remove whiteSmilingFace
- [x] remove apple

### LINKED ISSUE #152: Google logo ligatures and single letters are duplicated and outlines
The sources have two `Google.logo` and `Googlelogo` glyphs, same for the single letter variants. Only one seems to be used in the features. Even if subsetting removes one variant, the ligatures are both decomposed outlines, when they should probably be composites of the single letter varaints to save bytes.

### LINKED ISSUE #264: Remove duplicate logo forms
The following glyphs are duplicates of *.logo designs, are not defined in features, and can be removed from the glyph set:

- Glogo
- ologo
- glogo
- llogo
- elogo
- Googlelogo

### LINKED ISSUE #280: Add zero advance width control characters
Related:

- https://github.com/googlefonts/noto-fonts/issues/2036 
- https://github.com/googlefonts/fontbakery/issues/3205

We need to add the following control characters with zero adv width definitions:

```
0x00AD                            // SOFT HYPHEN
0x034F                            // COMBINING GRAPHEME JOINER
0x061C                            // ARABIC LETTER MARK
(0x200C <= c && c <= 0x200F)      // ZERO WIDTH NON-JOINER..RIGHT-TO-LEFT MARK
(0x202A <= c && c <= 0x202E)      // LEFT-TO-RIGHT EMBEDDING..RIGHT-TO-LEFT OVERRIDE
(0x2066 <= c && c <= 0x2069)      // LEFT-TO-RIGHT ISOLATE..POP DIRECTIONAL ISOLATE
0xFEFF                            // BYTE ORDER MARK
```

### LINKED ISSUE #308: Some Armenian glyphs have proprietary #anchors
Vo-arm and vew-arm.sc specifically have `#exit`, which is a Glyphs.app-specific way of marking component attachment points. Needs a review to see if the pipeline does the right thing.

### PR DESCRIPTION
**Pull request by madig**
Closes #122
Closes https://github.com/googlefonts/googlesans/issues/308
Closes https://github.com/googlefonts/googlesans/issues/152
Closes https://github.com/googlefonts/googlesans/issues/264
Closes https://github.com/googlefonts/googlesans/issues/280.

Also:

- cleans up [18] bracket layer directories

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Will wait on Gurmukhi and Georgian merges to merge this.

**Comment by madig** [comment]
TTX says no diff when #anchors are removed.

**Comment by chrissimpkins** [comment]
> TTX says no diff when #anchors are removed.

Good news!  Thanks Nikolaus!

**Comment by madig** [comment]
@chrissimpkins ping to let you know I axed the logo duplicates and declared the dot-logo variants the one source of truth.

**Comment by chrissimpkins** [comment]
Changes to address #122: LGTM
Logo forms tested with changes to address #152: LGTM

**Comment by chrissimpkins** [comment]
Changes to remove #exit Glyphs app specific anchors / address https://github.com/googlefonts/googlesans/issues/308: LGTM

File: qa/definitions/GoogleSans-Bold.ttf.glyphsetdef [hunk 1]
```diff
@@ -2966,4 +2966,18 @@ eShortMatra_anusvaradeva
 eShortMatra_candraBindudeva
 eShortMatra_rephdeva
 eShortMatra_reph_anusvaradeva
-reph_anusvaradeva
\ No newline at end of file
+reph_anusvaradeva
+uni034F
```
Comment (chrissimpkins) [inline qa/definitions/GoogleSans-Bold.ttf.glyphsetdef:2970]: It looks like the following from #280 are already in the glyph set:

- U+00ad (Soft hyphen)
- U+200C (zero width nonjoiner)
- U+200D (zero width joiner)

For some reason each of these invisible code points have defined outlines.  It won't save a tremendous amount of space but we may as well remove the outlines on those three glyphs as part of this merge.

**Review by chrissimpkins — CHANGES_REQUESTED** [review]
Thanks so much for all of these changes Nikolaus. This looks great.  One minor request to remove outlines from the #280 glyphs that were already in the set.  Otherwise we are gtg here.

File: qa/definitions/GoogleSans-Bold.ttf.glyphsetdef (hunk 1 above)
Comment (madig) [reply qa/definitions/GoogleSans-Bold.ttf.glyphsetdef:2970]: Done in https://github.com/googlefonts/googlesans/pull/307/commits/d6dc7accd60a01b4b5cc39e833d4db776eac3e49.

**Review by chrissimpkins — APPROVED** [review]
LGTM!  I'll update the changelog and merge this today.  Thank you very much Nikolaus!

**Comment by chrissimpkins** [comment]
Changelog updated in 5ef39aeb7257be42765eca644ac52cbfbb755f56

**Comment by chrissimpkins** [comment]
Thanks again Nikolaus!

**Pull request merged**